src/Avatar/Avatar.js

```diff
@@ -34,10 +34,18 @@ export const styles = (theme: Object) => ({
 
 type ProvidedProps = {
   classes: Object,
+  theme?: Object,
+};
+
+type DefaultProps = {
   component: ElementType,
 };
 
 export type Props = {
+  /**
+   * Other base element props.
+   */
+  [otherProp: string]: any,
```

Comment:
What is this?

Reply:
Since we are validating everything, we have to allow indexed (unsealed objects).  This allows properties to pass down through the render stack i.e. html properties.

Reply:
https://github.com/callemall/material-ui/issues/8716#issuecomment-340974033

Reply:
Ok, sounds good. I think that we miss `@ignore` not to have them documented.

Reply:
> As far as [otherProp: string]: any, if we are now strongly typing through the call stack and we have no way of importing Props from the underlying element (one to many) which we do not...then this is correct. You have to effectively have an unsealed set of props to allow pass through - so this makes perfect sense.

Hummm